### PR TITLE
DEVDOCS-4819:[update] channels.mdx

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4819]

## What changed?
Added limits to channels.mdx

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4819]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4819?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/headless/channels.mdx [hunk 1]
```diff
@@ -14,6 +14,11 @@ You can create both using the [Channels API](/docs/rest-management/channels).
 2. Pass that channel ID in a `POST` request to `/channels/{channel_id}/site` to create a site for the provided channel.
 3. After setting up the channel, you can authenticate cross-origin requests by [creating a JWT token](/docs/storefront-auth/tokens#create-a-token) through the GraphQL Storefront API. Make sure to create your GraphQL Storefront API token with the same channel ID as your headless platform; otherwise, your request will be rejected.
 
+<Callout type="info">
+#### Limits
+When the BigCommerce storefront status is `inactive` or `prelaunch`, one active headless storefront and a free store are allowed.
```
Comment (kristinapototska) [inline docs/api-docs/headless/channels.mdx:19]: I think we would need to add more details here.
We need to mention that each store comes with 1 free active storefront. If merchants want this storefront to be headless – they need to set default BC storefront to `inactive` status and from there they can create `active` headless storefront within that 1 free storefront seat that comes with a store.

File: docs/api-docs/headless/channels.mdx [hunk 2]
```diff
@@ -14,6 +14,11 @@ You can create both using the [Channels API](/docs/rest-management/channels).
 2. Pass that channel ID in a `POST` request to `/channels/{channel_id}/site` to create a site for the provided channel.
```
Comment (kristinapototska) [inline docs/api-docs/headless/channels.mdx:14]: We also need to add additional steps here, that I've mentioned in the ticket.
Where developers need to:
1. Create site for headless channel which is going to be given channel primary domain.
2. Create checkout domain for given headless channel using https://developer.bigcommerce.com/docs/rest-management/channels/channel-site-checkout-url#upsert-a-site's-checkout-url
3. Add 3P SSL (optional) – https://developer.bigcommerce.com/docs/rest-management/sites/site-certificate#upsert-a-site%E2%80%99s-ssl/tls-certificate-information

**Review by kristinapototska — CHANGES_REQUESTED** [review]

**Review by kristinapototska — APPROVED** [review]

File: docs/api-docs/headless/channels.mdx [hunk 3]
```diff
@@ -10,10 +10,18 @@ Before you can display products on a headless storefront, you need to create a c
...
+4. Give this channel the primary domain and set the checkout domain for the headless storefront by sending a `PUT` request to the `/channels/{channel_id}/site/checkout-url` endpoint.
+5. Optionally, you can add a 3P SSL certificate for the checkout domain by sending a `GET` request to the `/sites/{site_id}/certificate` endpoint. See [Upsert a Site’s SSL/TLS Certificate Information](/docs/rest-management/sites/site-certificate#upsert-a-site’s-ssl/tls-certificate-information) for details.
+6. After setting up the channel, you can authenticate cross-origin requests by [creating a JWT token](/docs/storefront-auth/tokens#create-a-token) through the GraphQL Storefront API. Create your GraphQL Storefront API token with the same channel ID as your headless platform; otherwise, your request will be rejected.
+
+<Callout type="info">
+#### Limits
+Every store comes with a default limit of one active storefront that comes for free along with a store. The store can be used as a free sandbox while it is in inactive status. If you want also to make the store active, you need to purchase an additional storefront seat.
```
Comment (bc-tgomez) [inline docs/api-docs/headless/channels.mdx:22]:
```suggestion
 Every store comes with a default limit of one active storefront that comes for free along with a store. The store can be used as a free sandbox while it is in inactive status. If you want also to make the store active, you need to purchase an additional storefront seat.
```

**Review by bc-tgomez — CHANGES_REQUESTED** [review]
The content changes look great! For the callout, we need to add a space before each sentence. [Nextra style guide for reference](https://mdx-components--bigcommerce-dev-center.netlify.app/components/callouts).

**Pull request merged**